src/script/user/UserRepository.ts

```diff
@@ -510,13 +511,14 @@ export class UserRepository {
       return Promise.resolve([]);
     }
 
-    const _getUsers = (chunkOfUserIds: string[]) => {
+    const _getUsers = (chunkOfUserIds: string[]): Promise<(void | User)[]> => {
```

Comment:
```suggestion
    const _getUsers = (chunkOfUserIds: string[]): Promise<Array<void | User>> => {
```

Reply:
I took the style from "mapUsersFromJson" to keep it consistent.